lib/index.ts

```diff
@@ -0,0 +1,10 @@
+import { types } from './types';
+export * from './tokenizer';
+import { tokenizer } from './tokenizer';
+export * from './types';
+
+export default tokenizer;
+export { types };
+
+module.exports = tokenizer;
+module.exports.types = types;
```

Comment:
interesting. so this is how you can integrate es6 imports while still supporting the old `module.exports`? i've been using a different syntax on other libraries, it looked like

```js
export = tokenizer
```

but it has several disadvantages. if your way is the correct way to do it, i have a bunch of other libraries to update.

Reply:
I haven't done a lot of work on libraries that have had to use old and new imports and only used this because I have seen it in passing.

You may want to look into it a little more before you go updating other libraries.